app/src/main/java/nerd/tuxmobil/fahrplan/congress/system/OnLocaleChangedReceiver.java

```diff
@@ -0,0 +1,34 @@
+package nerd.tuxmobil.fahrplan.congress.system;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+
+import org.ligi.tracedroid.logging.Log;
+
+import nerd.tuxmobil.fahrplan.congress.notifications.NotificationHelper;
+
+public final class OnLocaleChangedReceiver extends BroadcastReceiver {
+
+    private static final String LOG_TAG = "onLocaleChanged";
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        if (intent == null) {
+            return;
+        }
+        String action = intent.getAction();
+        if (action == null) {
+            return;
+        }
+        if (!action.equals(Intent.ACTION_LOCALE_CHANGED)) {
+            return;
+        }
+
+        Log.d(LOG_TAG, "onReceive (locale changed)");
```

Comment:
Please remove the log statement and the corresponding `LOG_TAG` and import.
Side node: `org.ligi.tracedroid.logging.Log` is used for Logs which become part of a crash report. Therefore only use them if you trying to trace down a bug. Use `android.util.Log` instead and switch on `MyApp#DEBUG` temporarily.

Reply:
I saw you changed the import in 3fa3217191e75ce614412947b95cb09160794cd9. But do we need to leave the log in the code at all?

Reply:
The initial Code was inspired by the already existing OnBootReceiver class. I am not sure what logging guide lines apply in this project. But generally speaking I think some kind of trace log, where each method entry is logged, is not too bad.

Reply:
I leave the logging in for the moment.

There is not logging guidelines written down for the project. I recommend using log statements sparsely and only if there is a particular reason for it such as debugging a feature or collection traces over a longer time. If too much logging is used then the logcat output becomes unreadable and less useful. Therefore, please do not log every method entry.